src/main/resources/zeebe_release.bpmn

```diff
@@ -218,7 +220,7 @@
     <bpmn:sequenceFlow id="Flow_12eezq7" name="Yes" sourceRef="Gateway_1afx4hl" targetRef="Gateway_113spke">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">= release_type = "major" or release_type = "minor"</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="Flow_1n61zbo" name="No" sourceRef="Gateway_1afx4hl" targetRef="Gateway_15n27wl" />
+    <bpmn:sequenceFlow id="Flow_1n61zbo" name="No - Alpha release" sourceRef="Gateway_1afx4hl" targetRef="Gateway_15n27wl" />
```

Comment:
:wrench: We should adjust the name. The path is taken for `alpha` and `patch` releases.